### ISSUE TITLE
The accuracy of validation during training RGB stream is basically unchanged

### ISSUE THREAD

**Issue by Abby-Wu** [craston/MARS#24]
Thank you for your good effort.

I want to ask that the accuracy of validation during training RGB stream is basically unchanged. I tried to fine-tune the RGB stream on UCF101 dataset. The accuracy of validation is stable at around 0.86 since step 40.  

Is my training process right?  I do not modify the hyperparameters.

**Comment by craston** [comment]
The validation accuracy is reported per clip whereas the test accuracy is averaged over all the clips in the video.
Validation clip accuracy of around 86% seems ok . You could check the test accuracy using this  trained model
